src/LatteTemplateResolver/NetteApplicationUIPresenterGlobals.php

```diff
@@ -25,30 +24,18 @@ protected function getClassGlobalVariables(ReflectionClass $reflectionClass): ar
     protected function getClassGlobalComponents(ReflectionClass $reflectionClass): array
     {
         $className = $reflectionClass->getName();
-        return array_merge(
-            $this->componentFinder->find($className, ''),
```

Comment:
do you remember why we had `''` here? :) I've removed it and it still works

Reply:
that is collected via $this->findInClasses($className) so methods are not required